api/grpcserver/v2alpha1/interface.go

```diff
@@ -0,0 +1,13 @@
+package v2alpha1
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+// genesisTimeAPI is an API to get genesis time and current layer of the system.
+type genesisTimeAPI interface {
+	GenesisTime() time.Time
```

Comment:
Does it make sense to have a method to get the genesis time? It's constant and could be passed to `NewNetworkService()` like other constants.